README.md

```diff
@@ -29,7 +29,7 @@ These notebooks are written for Python 2.7. To run them:
 ```
 pip install -r requirements.txt
 ```
-2. (optional: to skip training) Download the latest [model](https://storage.googleapis.com/unintended-ml-bias-analysis/models/wiki_tox_labels_v1_model.h5) and [tokenizer](https://storage.googleapis.com/unintended-ml-bias-analysis/models/wiki_tox_labels_v1_tokenizer.pkl) to the `models/` subdirectory.
+2. (optional: to skip training) Download the latest [model](https://storage.googleapis.com/unintended-ml-bias-analysis/models/[MODEL_NAME]_model.h5) and [tokenizer](https://storage.googleapis.com/unintended-ml-bias-analysis/models/[MODEL_NAME]_tokenizer.pkl) to the `models/` subdirectory.
```

Comment:
Is there a list of the valid `MODEL_NAME` values somewhere?

Reply:
Not yet. We've changed to training many models so will need to revisit this later.